cmd/types.go

```diff
@@ -49,18 +49,22 @@ type DockerComposeService struct {
 
 // ShipmentEnvironment represents a shipment/environment combination
 type ShipmentEnvironment struct {
-	Name           string             `json:"name,omitempty"`
-	EnvVars        []EnvVarPayload    `json:"envVars,omitempty"`
-	Ports          []PortPayload      `json:"ports,omitempty"`
-	Containers     []ContainerPayload `json:"containers,omitempty"`
-	Providers      []ProviderPayload  `json:"providers,omitempty"`
-	ParentShipment struct {
-		Name    string          `json:"name,omitempty"`
-		EnvVars []EnvVarPayload `json:"envVars,omitempty"`
-		Group   string          `json:"group,omitempty"`
-	}
-	BuildToken       string `json:"buildToken,omitempty"`
-	EnableMonitoring bool   `json:"enableMonitoring,omitempty"`
+	Username         string             `json:"username"`
+	Token            string             `json:"token"`
+	Name             string             `json:"name,omitempty"`
```

Comment:
Seems like we should remove `omitempty` for non-optional members?

Reply:
didn't want to change it how it already was. You think we should remove?